templates/students/attendance.html

```diff
@@ -6,145 +6,198 @@
+        {% else %}
+            <form method="POST">
+                {% csrf_token %}
+                <div class="row justify-content-center mt-5">
+                    <div class="col-sm-6 mb-2 text-center">
+                        <div class="row justify-content-center">
+                            <div class="col mb-4">
+                                <select name="selected_class" class="custom-select d-block w-100" id="stu-att-class">
+                                    <!-- <option value="all">Select Class</option> -->
+                                    <option value="1-3">Class 1-3</option>
+                                    <option value="4-5">Class 4-5</option>
+                                    <option value="1-5">Class 1-5</option>
+                                    <option value="6-8">Class 6-8</option>
+                                    <option value="1-8">Class 1-8</option>
+                                    <option value="9-10">Class 9-10</option>
+                                    <option value="13-14">Navodaya Wing</option>
+                                </select>
+                            </div>
+                        </div>
+                        <input type="text" id="filter-students" class="form-control mb-3 mx-auto"
+                               onkeyup="filterStudents()" placeholder="Search for names..">
+                        <table class="table" id="stu-att-table">
+                            <thead>
+                            <tr>
+                                <th>Student Name</th>
+                                <th>Class</th>
+                                <th>Mark Present</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            {% for stu_att in stu_att_today %}
+                                <tr>
+                                    <td>{{ stu_att.student.get_full_name }}</td>
+
+                                    {% if stu_att.student.school_class == 13 %}
+                                        <td>JNV_4</td>
+                                    {% elif stu_att.student.school_class == 14 %}
+                                        <td>JNV_5</td>
+                                    {% else %}
+                                        <td>{{ stu_att.student.school_class }}</td>
+                                    {% endif %}
+
+                                    <td>
+                                        <div class="custom-control custom-switch">
+                                            <input type="checkbox" name="attended" value="{{ stu_att.student.id }}"
+                                                   class="custom-control-input"
+                                                   id="s{{ stu_att.id }}"
+                                                   onchange="attendance({{ stu_att.student.id }}, {{ stu_att.id }})"
```

Comment:
You can pass the arguments as strings here. Also, pass the id of the checkbox instead of `stu_att.id`.

Try this: `onchange="attendance('{{ stu_att.student.id }}', 's{{ stu_att.id }}')"`